lib/cli.mjs

```diff
@@ -572,14 +517,14 @@ export default async function main(argv) {
 				process.stderr.write(output);
 			}
 
-			if (options.outputFile) {
-				writeOutputFile(output, options.outputFile).catch(handleError);
+			if (isString(outputFile)) {
+				writeOutputFile(output, outputFile).catch(handleError);
 			}
 
-			if (linted.errored) {
+			if (errored) {
 				process.exitCode = EXIT_CODE_ERROR;
-			} else if (maxWarnings !== undefined && linted.maxWarningsExceeded) {
-				const foundWarnings = linted.maxWarningsExceeded.foundWarnings;
+			} else if (isNumber(maxWarnings) && maxWarningsExceeded) {
```

Comment:
nit

Since you were using predicates everywhere else, I was expecting `isPlainObject(maxWarningsExceeded)`.

Reply:
This `isNumber(maxWarnings)` is needed for [narrowing](https://www.typescriptlang.org/docs/handbook/2/narrowing.html) the `maxWarnings` type from `unknown` to `number`. On the other hand, `maxWarningsExceeded` from `standalone()` already has the concrete type, so `isPlainObject()` is unnecessary.